app/components/ui/Translate.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Helmet } from 'react-helmet-async';
+
+// This is a prototype component for testing out the Google Translate widget. The logic below
+// dictates that this component only be displayed on staging; i.e. the widget will be displayed IF
+// location.hostname === 'staging.sfserviceguide.org'. To test locally point
+// staging.sfserviceguide.org to your local host IP in your /etc/hosts file; i.e.,
+// 127.0.0.1       staging.sfserviceguide.org
+// Then, navigate to http://staging.sfserviceguide.org:8080 to see the widget.
+//
+// Note: At least while testing, this is a <li> element as it goes within the
+// nav <ul> site links element.
+
+const Translate = () => {
+  const isStaging = window.location.hostname === 'staging.sfserviceguide.org';
+  if (isStaging) {
+    return (
+      <li className="googleTranslateContainer">
+        {/* todo: When we go live with Google Translate OR we have a dedicated test environment,
+                  the scripts within this Helmet snippet should be moved to index.html  */}
```

Comment:
It might still need to be gated based on the specific whitelabeled site.

Reply:
> A few small comments, but this is a pretty straightforward component, so this LGTM!

Good points, per usual. I've clarified/edited the comments a bit. Will merge in now.